test/extended/images/mongodb_replica.go

```diff
@@ -0,0 +1,114 @@
+package images
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/db"
+)
+
+var _ = g.Describe("[images][mongodb] openshift mongodb replication", func() {
+	defer g.GinkgoRecover()
+
+	const (
+		templatePath         = "https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb-clustered.json"
+		deploymentConfigName = "mongodb"
+		expectedValue        = `{ "status" : "passed" }`
+		insertCmd            = "db.bar.save(" + expectedValue + ")"
+	)
+
+	const (
+		expectedReplicasAfterDeployment = 3
```

Comment:
I'm fine with this and `deploymentConfigName` being copies of values from the template.

If this test runs for every PR in the mongodb repo, we should be fine as changing those values would break the test and it should be easy to fix (but perhaps impossible to do in lockstep?).